### ISSUE TITLE
:bug: Crashes if setting incorrect warning

### ISSUE THREAD

**Issue by edmundito** [AtomLinter/linter-coffeelint#55]
CoffeLint will crash completely if one accidentally puts the wrong warning level int he lint configuration. the linter will throw an exception that will cause coffelint to crash.

**Steps to Reproduce**
Use `warning` instead of `warn`:
```
"max_line_length": {
      "level": "warning",
      "value": 80
    }
```

**Expected**
An more user-friendly way to display the error based on the exception.
